### DISCUSSION TITLE
科技公司能把你的臉變成自家數據嗎？

### DISCUSSION THREAD

**Discussion by MAKERRRofKEVIN** [1101-GS4719-C/Python-Community-GS4719-C#201] [General]
2020寫的文章，順邊宣傳一下我們家的教育平台，談的是關於科技與人權的衝突，遭指控的公司是美國做社群網路人臉辨識及搜索應用的科技公司「Clearview AI」，指控其違法的組織則是全美數一數二的老牌左派組織ACLU，其歷史已愈百年，兩方的訴訟焦點在於網路上的公開資料是否能被科技公司在為經當事人許可的前提下用來尋人之用。
https://www.instagram.com/p/CDoE5KknW_q/

**Comment by Jen-Siang** [comment]
若能在這之中尋求平衡，相信對人們還是利大於弊的
